pipelines/main/misc/pdf_docs/build_pdf_docs.yml

```diff
@@ -0,0 +1,55 @@
+steps:
+  - group: "Check"
+    steps:
+      - label: "pdf_docs_build"
+        key: pdf_docs_build
+        # depends_on:
+        #   - "build_x86_64-linux-gnu"
+        plugins:
+          - JuliaCI/external-buildkite#v1:
+              version: "./.buildkite-external-version"
+              repo_url: "https://github.com/JuliaCI/julia-buildkite"
+          - JuliaCI/julia#v1:
+              # Drop default "registries" directory, so it is not persisted from execution to execution
+              persist_depot_dirs: packages,artifacts,compiled
+              version: '1'
+          - staticfloat/sandbox#v1:
+              rootfs_url: https://github.com/JuliaCI/rootfs-images/releases/download/v5.38/latex.x86_64.tar.gz
+              rootfs_treehash: "be2d12d899ee334c470a79f6d3f3faa4e63c7c3c"
+              uid: 1000
+              gid: 1000
+              workspaces:
+                # Include `/cache/repos` so that our `git` version introspection works.
+                - "/cache/repos:/cache/repos"
+        commands: |
+          # echo "--- Download pre-built julia, extract into usr/"
+          # buildkite-agent artifact download --step "build_x86_64-linux-gnu" 'julia-*-linux-x86_64.tar.gz' .
+          # mkdir -p ./usr
+          # tar -C ./usr --strip-components=1 -zxf julia-*-linux-x86_64.tar.gz
+          # ln -s ./usr/bin/julia ./julia
+
+          echo "--- Build Julia"
```

Comment:
Why does this need to build julia? Can't it use the existing binary from a build on the same trigger?

Reply:
Yes, it should. I was just running into weird errors when I tried that initially. (`make` was complaining that it couldn't find a Fortran compiler.)

So I switched to building Julia just to get this PR working quickly, so that we can merge it and start using Buildkite to deploy the PDF manual.

Reply:
We are now using a pre-built Julia.